tee-worker/local-setup/launch.py

```diff
@@ -266,30 +273,38 @@ def main(processes, config_path, parachain_type, log_config_path, offset, parach
     parser = argparse.ArgumentParser(
         description="Run a setup consisting of a node and some workers"
     )
-    parser.add_argument("config", type=str, help="Config for the node and workers")
+    parser.add_argument("--config", type=str, help="Config for the node and workers")
     parser.add_argument(
-        "parachain",
+        "--parachain",
         nargs="?",
         default="local-docker",
         type=str,
         help="Config for parachain selection: local-docker / local-binary / remote",
     )
     parser.add_argument(
-        "log_config_path",
+        "--log",
```

Comment:
Hmm that's actually not a so good name 😅 
I'd say for the long name we can sue `--log-config-path`, for the short name we can use `-l`

Reply:
Updated ✅